.github/workflows/test-browser.yml

```diff
@@ -0,0 +1,88 @@
+name: Run browser tests in Karma
+on:
+
+  # Runs for pushes and pull requests,
+  #  but if changes are config-only
+  push:
+    branches:
+      - master
+      - work-karma-ci
+    paths-ignore:
+      - 'src/config/**'
+  pull_request:
+    branches:
+      - master
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+    paths-ignore:
+      - 'src/config/**'
```

Comment:
@Gusted This is the filter I was referring to here:

https://github.com/darkreader/darkreader/pull/8122#issuecomment-1030570192